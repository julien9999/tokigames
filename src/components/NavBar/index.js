import React from 'react';

export default function NavBar() {
  return (
    <nav class="navbar navbar-light bg-light">
        <a class="navbar-brand" href="#">
            <img 
              src="https://media.glassdoor.com/sqll/2595586/tokigames-squarelogo-1558486351137.png" 
              style={{ marginRight: "10px" }}
              width="30" 
              height="30" 
              class="d-inline-block align-top" 
              alt="" 
            />
            Tokigames-test
        </a>
    </nav>
  );
}

