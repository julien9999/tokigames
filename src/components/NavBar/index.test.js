import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from './index';


describe('<NavBar>', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <NavBar />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

