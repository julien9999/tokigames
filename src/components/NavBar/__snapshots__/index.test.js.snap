// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar> renders correctly 1`] = `
<nav
  class="navbar navbar-light bg-light"
>
  <a
    class="navbar-brand"
    href="#"
  >
    <img
      alt=""
      class="d-inline-block align-top"
      height="30"
      src="https://media.glassdoor.com/sqll/2595586/tokigames-squarelogo-1558486351137.png"
      style={
        Object {
          "marginRight": "10px",
        }
      }
      width="30"
    />
    Tokigames-test
  </a>
</nav>
`;
